### DISCUSSION TITLE
Donation/ paid version

### DISCUSSION THREAD

**Discussion by Jazzmega** [jgraph/drawio#2584] [General]
Hello, and thanks for this magnificent tool.

How can we donate to you or buy a paid version?

I think it's fair to pay after this work.

Thanks again

**Comment by Peeeet** [comment]
You can sponsor "JGraph" at https://github.com/sponsors/jgraph#sponsors if you want, but I think that's the most you can do aside from getting a confluence /  jira subscription as indirect support since I think Diagrams.net is mostly financially supported by those integrations.
